src/core/server/base.js

```diff
@@ -83,13 +84,26 @@ export default function(routes, createStore) {
         );
 
         const componentHTML = renderToString(InitialComponent);
+
         const assets = webpackIsomorphicTools.assets();
-        const styles = Object.keys(assets.styles).map((style) =>
-          `<link href="${assets.styles[style]}" rel="stylesheet" type="text/css" />`
-        ).join('\n');
-        const script = Object.keys(assets.javascript).map((js) =>
-          `<script src="${assets.javascript[js]}"></script>`
-        ).join('\n');
+
+        // Get SRI for production only.
+        const sri = (ENV === 'production') ? JSON.parse(
+          fs.readFileSync(path.join(config.get('basePath'), 'sri.json'))
+        ) : null;
+
+        const styles = Object.keys(assets.styles).map((style) => {
+          const cssSRI = sri && ` integrity="${sri[path.basename(assets.styles[style])]}"
```

Comment:
Should we throw an error if `sri[path.basename(assets.styles[style])]` is `undefined`?